### PR TITLE
DOCSP-38680: v2.25 updates

### PR DESCRIPTION
**Pull request by ccho-mongodb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-38680>
Staging:
- [Compatibility](https://preview-mongodbcchomongodb.gatsbyjs.io/csharp/DOCSP-38680-v2.25/compatibility/)
- [What's New](https://preview-mongodbcchomongodb.gatsbyjs.io/csharp/DOCSP-38680-v2.25/whats-new/#what-s-new-in-2.25)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/includes/language-compatibility-table-csharp.rst [hunk 1]
```diff
@@ -10,6 +10,13 @@
      - .NET 4.8
      - .NET 4.7 [#2.14-note]_
      - .NET 4.6
+   * - 2.25
```
Comment (mongoKart) [inline source/includes/language-compatibility-table-csharp.rst:13]: Suggestion: Condense rows 2.20 through 2.25 to a single row, similar to Go.

Comment (ccho-mongodb) [reply source/includes/language-compatibility-table-csharp.rst]: Looks like 2.14 through 2.25 are the same, so I'll do it for those.

File: source/whats-new.txt [hunk 2]
```diff
@@ -28,6 +29,25 @@ Learn what's new in:
...
+- Added support for the ``MONGODB-OIDC`` authentication mechanism and automatic
+  token acquisition for Azure Identity Provider.
+- Added the class name to the error message reported when
+  ``BsonClassMapSerializer`` cannot locate a matching creator.
+- Added a ``LoggedStages`` field to surface the MQL executed after performing
+  a LINQ query.
+- Added support for overriding the ``"mongodb"`` service name with a custom one
+  by using the ``srvServiceName`` connection option.
+- Fixed behavior in which bulk write operations enumerated the requests
+  parameter more than once.
```
Comment (BorisDog) [inline source/whats-new.txt:49]: I think 'Improved' instead of 'fixed' is better, as it technically it was not wrong.

**Review by BorisDog — CHANGES_REQUESTED** [review]
minor comment

**Review by BorisDog — APPROVED** [review]
LGTM

**Pull request merged**